### PR TITLE
Intro rewrite

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
First pass on rewriting the intro.

### CODE REVIEW

File: contributing/adding_datasets.md [hunk 1]
```diff
@@ -9,232 +9,109 @@ parent: Contributing to Data Commons
...
+Follow the template at <https://github.com/datacommonsorg/data/tree/master/scripts/example_provenance/example_dataset> to construct a README for your dataset.
 
-#### Step 2b: Check In the Preprocessing Script.
+#### Final steps
 
-Check in the script and the resulting CSV to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/covid_tracking_project/historic_state_data).
+Check in the Template MCF file together with the cleaned CSV, its preprocessing script (if needed), and the README to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/fred/homeownership). If you wrote a script to automate the generation of the TMCF, you may also include that.
```
Comment (beets) [inline contributing/adding_datasets.md:100]: Instead of "check in", should we say "send a pull request"? Not sure what the accepted term is for github.

**Review by beets — CHANGES_REQUESTED** [review]
This is nice! Sorry for the late review, I just have a few small comments.

File: contributing/adding_datasets.md [hunk 2]
```diff
@@ -9,232 +9,109 @@ parent: Contributing to Data Commons
...
 
-|country_id  |  country_name	         |  continent_id|
-|-------|--------|---------|
-|USA	     |  United States of America |  northamerica|
-|IND	     |  India                    |	        asia|
+Sometimes the CSV needs to be cleaned before it can be used. There are no restrictions on your approach for this step, only requirements for its result.
 
+1. Each `StatisticalVariable` must have its own column for its observed value.
+2. Each property in your schema must have its own column for its value, including the values of `observationAbout` and `observationDate`.
+3. Dates must be in ISO 8601 format: "YYYY-MM-DD".
```
[the remark this answers is not in the document]
Comment (tjann) [reply contributing/adding_datasets.md:22]: Technically no, but we strongly prefer as long as the obs period is obvious

File: contributing/adding_datasets.md [hunk 3]
```diff
@@ -9,232 +9,109 @@ parent: Contributing to Data Commons
...
-
-## Conclusion
 
-After producing all three required components (`StatisticalVariable` schema MCF, cleaned CSV, and Template MCF), go ahead and submit a PR to [https://github.com/datacommonsorg/data/pulls](https://github.com/datacommonsorg/data/pulls).
\ No newline at end of file
+Once you've generated the MCF, write a README following the template at <https://github.com/datacommonsorg/data/tree/master/scripts/example_provenance/example_dataset>. Finally, check in the MCF file together with any script used to assemble it and the README to [https://github.com/datacommonsorg/data](https://github.com/datacommonsorg/data) under the appropriate [`scripts/<provenance>/<dataset>` subdirectory](https://github.com/datacommonsorg/data/tree/master/scripts/covid_tracking_project/historic_state_data).
```
Comment (tjann) [inline contributing/adding_datasets.md:117]: Maybe use this link for second link: https://github.com/datacommonsorg/data/tree/master/scripts/example_provenance/example_dataset

**Review by tjann — APPROVED** [review]

**Review by beets — APPROVED** [review]

**Comment by beets** [comment]
Thanks Anne - I'll merge this in and work on the suggested fixes from this PR as a follow up

**Pull request merged**